src/openrct2-ui/audio/AudioContext.cpp

```diff
@@ -44,9 +44,9 @@ namespace OpenRCT2::Audio
             return _audioMixer;
         }
 
-        std::vector<std::string> GetOutputDevices() override
+        std::deque<std::string> GetOutputDevices() override
```

Comment:
Stick to vector this. This is only used in like one, two places it's only ever about 2 elements big.